src/pages/CreateFeedback.tsx

```diff
@@ -1,44 +1,65 @@
+import { useForm, SubmitHandler, Controller } from 'react-hook-form';
+import Select from 'react-select';
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 import { useState } from 'react';
-import { ProductRequest } from '../types';
-import { useNavigate, Link } from 'react-router-dom';
-import arrowDown from '../assets/shared/icon-arrow-down.svg';
-import arrowUp from '../assets/shared/icon-arrow-up.svg';
+
+// SVG
 import leftArrow from '../assets/shared/icon-arrow-left.svg';
 import newFeedback from '../assets/shared/icon-new-feedback.svg';
-import { useForm, SubmitHandler } from 'react-hook-form';
-interface IProps {
-  productRequests: ProductRequest[];
-}
+import check from '../assets/shared/icon-check.svg';
+
+// Types
+import { ProductRequest, LabelValue } from '../types';
+import { colorstyles } from './colorstyle';
 
 type Inputs = {
-  titleRequired: string;
-  feedbackRequired: string;
+  title: string;
+  feedback: string;
   category: string;
 };
 
+const categoryOptions: LabelValue[] = [
+  {
+    value: 'Enhancement',
+    label: 'Enhancement',
+  },
+  { value: 'Bug', label: 'Bug' },
+  { value: 'Feature', label: 'Feature' },
+  { value: 'UI', label: 'UI' },
+  { value: 'UX', label: 'UX' },
+];
+
+interface IProps {
+  productRequests: ProductRequest[];
+}
+
 export const CreateFeedback = (props: IProps) => {
   const { productRequests } = props;
   const navigate = useNavigate();
+  const [isSearchable, setIsSearchable] = useState(false);
```

Comment:
Suggestion to add some generic types for `useState`

```
const [isSearchable, setIsSearchable] = useState<boolean>(false);
```

Reply:
Updated